docs/developer/OPERATIONS.md

```diff
@@ -44,30 +57,65 @@ git checkout -b release-X.Y.Z-prep
 scripts/update_release.sh X.Y.Z
 ```
 
-This script will also change some things in the operations repository, those should
-also be submitted via a PR to the appropriate repository and merged after tag creation.
-
-Check and eventually update [CHANGELOG](../CHANGELOG.md).
+Check and eventually update [CHANGELOG](../CHANGELOG.md). If everything looks OK
 
 ```bash
 git commit -am 'Update version to X.Y.Z'
 git push -u origin release-X.Y.Z-prep
 ```
 
-And then submit `release-X.Y.Z-prep` as a PR against the `release-X.Y.Z` branch.
+and submit `release-X.Y.Z-prep` as a PR against the `release-X.Y.Z` branch.
+After the review is successfully finished and the PR is merged, the release branch is ready.
+You can continue with [UAT](###UAT) phase.
+
+### UAT
+
+Deploy the `release-X.Y.Z` branch to the **UAT** environment - the details on
+the deployment procedure can be found in the OPS repository documentation.
+Announce the upcoming release to the users and ask them to test their
+workflows or integrations using the **UAT** environment.
+
+> [!IMPORTANT]
+> In the case of major release it is necessary to namely announce all the breaking
+> changes and expressly ask the owners of all the known integrations to account for
```

Comment:
I've had the thought from time to time that we need to start tracking who these owners are. Should we compile a list of approvals needed for major releases?

Reply:
@jvaught-rh good point! They are currently OSIM, SDEngine, and osidb-bindings on our side and then CPaaS and Konflux outside of our team. I do not have any official contact information.

Reply:
Let me add this to confluence as it is also internal and likely not useful in a public repo.